README.md

```diff
@@ -84,7 +84,9 @@ If your application is running outside of Google Cloud Platform, such as locally
   2. Copy the key somewhere your application can access it. Be sure not to expose the key publicly.
   3. Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the full path to the key. The debug agent will automatically look for this environment variable.
```

Comment:
I think authentication options set in `config` should only be used as an explicit override in case users don't want to use application-default credentials. Going with the idea that the library should be as simple to use as possible, the config object should be minimal (and ideally, unnecessary at all).

This [document](https://google-cloud-python.readthedocs.io/en/latest/google-cloud-auth.html) (from the Python google-cloud) also seems to suggest that explicit credentials should be reserved for specific use cases. The Node.js version doesn't mention `GOOGLE_APPLICATION_CREDENTIALS` anywhere.